packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Quickstart
+
+Create an issuer config with your secret key. This config will be used to interact with the idOS.
+```js
+import { createIssuerConfig } from "@idos-network/idos-issuer-sdk-js";
+import { Wallet } from "ethers";
+
+const wallet = new Wallet("YOUR_PRIVATE_KEY");
+
+const issuerConfig = await createIssuerConfig({
+  nodeUrl: "https://node.url",
```

Comment:
I'm a bit afraid this might be confusing for readers, and they might think it's the RPC for the network they're using.

I don't think we need a lot of disambiguation here. Prolly just mentioning a better example value is enough.

```suggestion
  nodeUrl: "https://nodes.idos.network/", // or nodes.playground.idos.network
```